content/en/docs/setup/independent/troubleshooting-kubeadm.md

```diff
@@ -261,4 +261,23 @@ sudo yum-config-manager --add-repo https://download.docker.com/linux/centos/dock
 yum install docker-ce-18.06.1.ce-3.el7.x86_64
 ```
 
+## Not possible to pass a comma separated list of values to arguments inside a `--component-extra-args` flag
+
+`kubeadm init` flags such as `--component-extra-args` allow you to pass custom arguments to a control-plane
```

Comment:
"flags such as..." Are there other `kubeadm init` flags with the same limitations?
If there are we should change the heading and list them. If there aren't, I'd change this sentence to:

> The `kubeadm init` flag `--component-extra-args` allows...

or something similar

Reply:
i'm not aware of other flags that have the same problem, but `--component-extra-args` implies `--apiserver-extra-args`, `--controller-manager-extra-args` or `--scheduler-extra-args`. where "component" replaces one of the above.

> The kubeadm init flag --component-extra-args allows

hm, could you please elaborate on the idea? its more of a case where the flag does not allow the values in question.

Reply:
Ah, got it. I'll play around with on the docs smoke test page to see what it takes to make that render. But that can be for a future PR. Thanks for the quick turn around/explanation!